.circleci/config.yml

```diff
@@ -54,3 +63,65 @@ jobs:
       - store_artifacts: *artifact_env_min
       - save_cache: *save-node-modules-cache
       - save_cache: *save-yarn-cache
+  test262:
+    working_directory: ~/babel
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - restore-cache: *restore-yarn-cache
+      - restore-cache: *restore-node-modules-cache
+      - run:
+          name: Build Babel
+          command: BABEL_ENV=test make bootstrap
+      - run:
+          name: Link Babel
+          command: |
+            cd packages
+            for package in */; do
+              cd $package
+              yarn link
+              cd ..
+            done
+      - run:
+          name: Setup Test Runner
+          command: |
+            git clone --recurse-submodules https://github.com/babel/babel-test262-runner
+            cd babel-test262-runner
+            yarn
+            yarn add tap-mocha-reporter --dev
+            curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > jq
+            chmod +x ./jq
+            for package in ../packages/*/package.json; do
+              yarn link $(./jq -j ".name" $package)
+            done
+            node lib/download-node
+      - run:
+          name: Run Test262
+          command: node lib/run-tests I_AM_SURE | tee ~/test262.tap
```

Comment:
Rather than linking every Babel package, what do you think of adding a `--babelMonorepo ..` cli clag to `lib/run-tests`?

Reply:
That is something I wanted to do in a followup PR in which I want to:
1. Use something like yargs: http://yargs.js.org/
2. Refactor existing cli arguments
3. Add path to babel argument
4. Add path to whitelist files so that they can remain colocated to babel/babel repo
5. Add babel plugins/config argument